content/visualization/visualization_1.md

```diff
@@ -9,7 +9,7 @@ Streamlit uses Python to build shareable dashboards without the need for front-e
 Run the command below to download the example dashboard:
 
 ```bash
-git clone --depth 1  --filter=blob:none --sparse https://github.com/snowplow-incubator/snowplow-accelerator-resources.git ; 
+git clone --depth 1  --filter=blob:none --sparse https://github.com/snowplow-incubator/snowplow-accelerator-resources.git ;
 cd snowplow-accelerator-resources
 git sparse-checkout set enhanced-ecommerce-accelerator/snowflake/streamlit
```

Comment:
How should flag the different options in this command?

Reply:
Added tabs like in the docs